.github/workflows/stage.yml

```
@@ -53,17 +53,19 @@ jobs:
         run: |
           python3 -m pip install docker
 
-      - name: Log in to GHCR
-        run: |
-          echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+      - name: Log into registry ghcr.io
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d    # v3.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build and stage awx-operator
-        working-directory: awx-operator
+      - name: Stage awx-operator
         run: |
           BUILD_ARGS="--build-arg DEFAULT_AWX_VERSION=${{ github.event.inputs.default_awx_version }} \
-                      --build-arg OPERATOR_VERSION=${{ github.event.inputs.version }}" \
-          IMAGE_TAG_BASE=ghcr.io/${{ github.repository_owner }}/awx-operator \
-          VERSION=${{ github.event.inputs.version }} make docker-build docker-push
+              --build-arg OPERATOR_VERSION=${{ github.event.inputs.version }}" \
+          IMG=ghcr.io/${{ github.repository_owner }}/${{ github.repository }}:${{ github.event.inputs.version }} \
```

```suggestion
          IMG=ghcr.io/${{ github.repository }}:${{ github.event.inputs.version }} \
```

We need to remove `${{ github.repository_owner }}` here because it is already there from when it is set during the checkout:
* https://github.com/ansible/awx-operator/pull/1681/files#diff-ca6f66a88a8c14e7412271d092d23b124f23052eb38134788e5a992f0522f574R49

---

https://github.com/ansible/awx-operator/blob/4d7936515c89ab9c0be6fb985fd968640874bf26/.github/workflows/stage.yml#L46-L50

I did not know this, however note taken.

question, re variable preceedence

https://github.com/ansible/awx-operator/blob/4d7936515c89ab9c0be6fb985fd968640874bf26/.github/workflows/stage.yml#L40-L50

there are two checkouts, which variable takes precedence? first set, last set?